### PR TITLE
merge prometheus 

### PR DESCRIPTION
**Pull request by xgao3**
HI Kenny, can you see if new content meets your requirements

Thanks.

--xhg.

### CODE REVIEW

File: prometheus/README.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+prometheus-kube-state-metrics             ClusterIP      10.100.200.54    <none>                     8080/TCP            1d
+prometheus-operated                       ClusterIP      None             <none>                     9090/TCP            1d
+prometheus-prometheus-node-exporter       ClusterIP      10.100.200.196   <none>                     9100/TCP            1d
+prometheus-prometheus-oper-alertmanager   ClusterIP      10.100.200.189   <none>                     9093/TCP            1d
+prometheus-prometheus-oper-operator       ClusterIP      10.100.200.137   <none>                     8080/TCP            1d
+prometheus-prometheus-oper-prometheus     ClusterIP      10.100.200.109   <none>                     9090/TCP            1d
+```
+**Note**:  Prometheus services do not have External IPs Mapped.
+
+* Outside of Grafana, Prometheus services are not accessible outsidee of the cluster.  If you want to reach Prometheus externally (from your desktop as an example), you can use port forwarding
```
Comment (kacole2) [inline prometheus/README.md]: typo on `outsidee` should be `outside`

File: prometheus/README.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+* Install a nginx ingress controller using helm
+
+    `helm install stable/nginx-ingress --name quickstart`
+    
+* We will deploy Promethus components in the monitoring namespace.  Let's create a monitoring namespace : 
+
+    `kubectl create namespace monitoring`  
+    
+  
+* Install Prometheus Operator using helm chart and reference the custom.yml
```
Comment (kacole2) [inline prometheus/README.md]: where did the `custom.yml` come from? I don't see that `yml` file here in these directions

File: prometheus/README.md (hunk 1 above)
Comment (kacole2) [reply prometheus/README.md]: this has been fixed

File: prometheus/README.md (hunk 2 above)
Comment (kacole2) [reply prometheus/README.md]: @xgao3 i'm not connecting the dots properly. where is `custom.yml` coming from?

Comment (xgao3) [reply prometheus/README.md]: fix the spelling mistake and also added the description for custom.yml

Comment (kacole2) [reply prometheus/README.md]: i see it now. cool! nice job. Can you squash your commits?

**Pull request merged**